jest.server.json

```diff
@@ -0,0 +1,17 @@
+{
+  "coverageDirectory": "./build2/reports/coverage",
+  "restoreMocks": true,
+  "moduleNameMapper": {
+    "^OktaAuth(.*)$": "<rootDir>/jquery/$1"
+  },
+  "testMatch": [
+    "**/test/spec/*.js"
+  ],
+  "testPathIgnorePatterns": [
+    "./test/spec/oauthUtil.js"
```

Comment:
nit:
it might be better to show coverage for all files so that will be crystal clear some files are not covered and why.